### ISSUE TITLE
Cannot import components from @primer/components

### ISSUE THREAD

**Issue by ghost** [primer/react#1010]
**Describe the bug**
Cannot import components from `@primer/components`.

**To Reproduce**
1. `npx create-react-app my-app`
2. Click on `src/App.js`
3. Add `import { Header } from "@primer/components";`
4. See error (Cannot resolve symbol `Header`)

**Expected behavior**
The component is imported through `@primer/component` in the [docs](https://primer.style/components/getting-started#installation), but it doesn't have any exports.

![image](https://user-images.githubusercontent.com/66206863/106707161-ce37a900-6616-11eb-8019-59e7bceff27b.png)

**Screenshots**

![image](https://user-images.githubusercontent.com/66206863/106706914-63866d80-6616-11eb-996c-2fc385b5dfac.png)

**IDE**
 - WebStorm v2020.3

**Comment by emplums** [comment]
@harshcut can you provide the output of `npm ls @primer/components` (or `yarn list @primer/components` if you're using yarn)?

**Comment by harshcut** [comment]
> @harshcut can you provide the output of `npm ls @primer/components` (or `yarn list @primer/components` if you're using yarn)?

![image](https://user-images.githubusercontent.com/66206863/106784236-1e902480-6672-11eb-8e5e-3601e0991904.png)

**Comment by emplums** [comment]
Hm that's strange @harshcut! I just updated another project with 23.0.2 and am not seeing this issue. If you can repro this in a code sandbox or other sharable link I can dig in deeper this week

**Comment by harshcut** [comment]
There seems to be no problem at all while importing in VSCode. Unfortunately, after switching back to WebStorm the problem still exists, even though the components are rendered.

#### Inspection info (WebStorm)

```
Cannot resolve symbol 'Button' 
Inspection info: Validates names and paths in ECMAScript import declarations.
```

**Comment by mathiasbosman** [comment]
The same happens in IntelliJ. 
The code compiles fine and runs ok when importing manually but the warning stays.

Also when adding a competent before importing it there's no 'hint' or smart importing available.

The exports don't seem to be picked up.

**Comment by MrPeker** [comment]
I have the same problem. I tried with Webstorm and Intellij Idea Ultimate and the problem still exists. But I have no problem in VS Code. Also, I upgraded to 24.x and the bug still exists.

**Comment by suneettipirneni** [comment]
I am also experiencing this same issue in WebStorm 2020.3.3, maybe it could be related to the same issue that FontAwesome described here: https://github.com/FortAwesome/vue-fontawesome/issues/177

**Comment by colebemis** [comment]
Is this still an issue?

**Comment by mathiasbosman** [comment]
It's been solved in #1136

**Comment by siddharthkp** [comment]
Looks like this issue was solved :)
